### ISSUE TITLE
PayStackDotNetSDK 

### ISSUE THREAD

**Issue by Xmuhad** [fzany/PayStackDotNetSDK#1]
PayStackDotNetSDK to create a transfer recipient using  CreateTransferRecipient method  it returning status waiting for activation

**Comment by fzany** [comment]
I would look into it right away.


Regards

Sent from Mail for Windows 10

From: Xmuhad
Sent: Wednesday, November 28, 2018 1:44 PM
To: fzany/PayStackDotNetSDK
Cc: Subscribed
Subject: [fzany/PayStackDotNetSDK] PayStackDotNetSDK (#1)

PayStackDotNetSDK to create a transfer recipient using CreateTransferRecipient method it returning status waiting for activation

**Comment by fzany** [comment]
> PayStackDotNetSDK to create a transfer recipient using CreateTransferRecipient method it returning status waiting for activation

Dear @Xmuhad ,
I have just tried to create a New transfer Recipient using the library and it's working all fine. 

Kindly perform these checks: 
1) Your Paystack account is valid and activated via Email.
2) Your Paystack account is not blocked or disabled.
3) Your Api secret is valid and not disabled. 
4) Your Nuget Package https://www.nuget.org/packages/PayStackDotNetSDK/ is updated.

Let me know the results of these. 
I just updated the repo with a new Project (Console1) which there is a quick code to create a new transfer recipient. 


Here is a screenshot of my test just a few minutes ago: 
![paycap](https://user-images.githubusercontent.com/20480611/49153827-1ceea780-f317-11e8-93f0-b8caaa02abb2.PNG)


Regards
